pkg/storage/replica_rangefeed.go

```diff
@@ -464,6 +475,9 @@ func (r *Replica) handleClosedTimestampUpdateRaftMuLocked(ctx context.Context) {
 	}
 }
 
+// ensureClosedTimestampStarted does its best to make sure that this node is
+// receiving closed timestamp updated for this replica's range. Note that this
+// forces a lease to exist somewhere and so is reasonably expensive.
```

Comment:
nit: I'd change "lease to exist somewhere and so is reasonably expensive" to "valid lease to exist on the range and so can be reasonably expensive if one is not already valid".